stoplight.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function () {
+  //Getting a referenceError abut $ is not defined. Why?
+
+let light = $(".bulb")
+function lightup(){
+  console.log("I was clicked")
+}
+.bulb.on("click", lightup).addClass(".active#red")
```

Comment:
- `.addClass` needs to be called on a jQuery object. Currently you are calling it directly on an event listener, which will not work.
- You only need to add the active class in this example. The divs already have their respective color ids in the HTML. You also would not use `.addClass` to add an id.
- When using `.addClass`, you do not need to precede the class name with a `.`. The method assumes that what you are passing in is a class.
- It looks like you have an extra `.` at the start of the line.